src/Sylius/Bundle/CoreBundle/Command/InstallDatabaseCommand.php

```diff
@@ -45,10 +47,13 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             $this->getEnvironment(),
         ));
 
+        $questionHelper = $this->getHelper('question');
```

Comment:
`phpstan-symfony` extension doesn't recognize this case?

Reply:
It looks good to me, the annotation says that it can be any HelperInterface implementation, but not QuestionHelper instances.